core/src/test/java/org/testcontainers/junit/DockerComposeServiceTest.java

```diff
@@ -0,0 +1,33 @@
+package org.testcontainers.junit;
+
+import org.junit.Rule;
+import org.junit.Test;
+import org.testcontainers.containers.DockerComposeContainer;
+
+import static org.rnorth.visibleassertions.VisibleAssertions.assertNotNull;
+
+import java.io.File;
+
+public class DockerComposeServiceTest extends BaseDockerComposeTest {
+
+    @Rule
+    public DockerComposeContainer environment = new DockerComposeContainer(new File("src/test/resources/compose-test.yml"))
+            .withServices("redis")
+            .withExposedService("redis_1", REDIS_PORT);
+
+
+    @Override
+    protected DockerComposeContainer getEnvironment() {
+        return environment;
+    }
+
+    @Test(expected = NullPointerException.class)
```

Comment:
You've found something interesting here - I wonder if `NullPointerException` is the best exception that Testcontainers could be throwing. We could change this, although that can be a separate improvement. 

WDYT?

Reply:
So Map this kind of exception to a custom one? Would be nice, due to error searching :)

Reply:
Yeah - there's probably a missing null-check in our code. Let's tackle this separately though.

Reply:
Ok, fine. Made a new Branch as child of this one.